libs/remotes/src/lib/prompt/prompt-service.ts

```diff
@@ -120,7 +121,7 @@ export const promptService = {
 
       // istanbul ignore next
       if (error instanceof AxiosError) {
-        reason = error.response?.data?.reason;
+        reason = errorToAxiosError(error).response?.data?.reason;
```

Comment:
We should be able to limit new branches from `?` to help with our test coverage since we have specified the data here. You might be able to write this with no question marks `reason = errorToAxiosError(error).response.data.reason;` but I'm not 100% sure.

```suggestion
        reason = errorToAxiosError(error).response?.data.reason;
```